apis/bulkupload/bulkserviceapi.yaml

```diff
@@ -57,397 +74,903 @@ paths:
                   type: string
-                  description: "This is a .csv file.\n  - Each line of the file is\
-                    \ a data record. Each record consists of one or more user fields,\
-                    \ separated by commas. Each successfully created user is associated\
-                    \ with the root organization of the org administrator.\nThe CSV\
-                    \ file format supported is as follows:\n  - NAME: Name of user(firstName+lastName)\n\
-                    \  - PHONE: Phone number. Either phone or email is mandatory.\n\
-                    \  - EMAIL: Email. Either phone or email is mandatory.\n  - Ext\
-                    \ Org id: The organisation ID to which the user must be associated(extOrgId).\
-                    \ This is an optional field. If a value is is provided, the user\
-                    \ is associated with that organisation, i.e. the sub-organization\
-                    \ of the root organization. If not, the user is only associated\
-                    \ to the root organization od the org administrator uploading\
-                    \ the users.\n  - Input Status: This is a mandatory attribute.\
-                    \ It indicates if the user is active or inactive in the organization.\n\
-                    \  - Ext User ID: This is a mandatory field. It is the ID provided\
-                    \ by the root or sub-organization to users to uniquely identify\
-                    \ them. "
-                  format: binary
-        required: true
+                  description: >-
+                    - This is a .csv file.
+                      - Each line of the file is a data record. Each record consists of one or more user fields, separated by commas. Each successfully created user is associated with the root organization of the org administrator.
+                    - The CSV file format for shadow-upload supported is as follows:
+                      - NAME: Name of user(firstName+lastName)
+                      - PHONE: Phone number. Either phone or email is mandatory.
+                      - EMAIL: Email. Either phone or email is mandatory.
+                      - Ext Org id: The organisation ID to which the user must be associated(extOrgId). This is an optional field. If a value is is provided, the user is associated with that organisation, i.e. the sub-organization of the root organization. If not, the user is only associated to the root organization od the org administrator uploading the users.
+                      - Input Status: This is a mandatory attribute. It indicates if the user is active or inactive in the organization.
+                      - Ext User ID: This is a mandatory field. It is the ID provided by the root or sub-organization to users to uniquely identify them.
+                    - The CSV file format for self-declared-user-upload supported is as follows:
+                      - Diksha UUID: It is the identifier of the user.
```

Comment:
@Hari-stackroute please check if we are adding DIKSHA specific parameter names to Sunbird.

Reply:
@Sunitashelar you are right, but as of now while generating the self-declared user report it will contain the Diksha specific parameter and the same generated report is uploaded for processing. So it should be there and more over it is mandatory parameter, I will ask Rahul to give suggestion on this.

Reply:
OK